content/md/assets/from-PAM-to-new-AM.md

```diff
@@ -0,0 +1,96 @@
+---
+id: from-PAM-to-new-AM
+themes: manage-your-assets
+title: From PAM to new Asset Manager
+popular: false
+ee-only: true
+related: upload-assets, assets-transformation
+---
+
+# From non-structured assets to structured ones
+
+You may know our famous PAM, Product Asset Manager. If not, just a short laius. Since 2015, Akeneo PIM has enabled you to manage assets thanks to the PAM. But the way we managed assets was quite different than with our brand new Asset Manager. Let's see why.
+
+An "old PAM asset" was a non-structured object which had some particularities.
+
+## All the assets had the same properties
+In the PAM, it was impossible to add attributes to the assets. They all had the same properties (a code, a description, tags, an end of use date).
+
+## The assets could be localisable but not their values
+It was possible to create localisable assets. In fact, only the file was localisable (meaning that you could have one file per locale), but not the values of the properties. For example, if you had 2 activated locales, you could have 2 different files but they would have the same description, tags, etc.
+
+## The assets had mandatory transformations per channel
+One very important thing in the PAM was the transformations feature. By default, all the assets had different variations per channel. You imported `reference files` and a `variation` per channel was automatically created. If you wanted to use assets in your product sheets, you had to define these transformations in order to get a variation per channel. You never used the reference file in your product forms.    
+
+## The `assets collection` product attribute could not have different values per locale/channel.
+It was totally impossible to define a value per locale neither per channel on this specific product attribute type. The localization was hold by the asset itself.
```

Comment:
replace "neither" with "nor"
the localization was "held" by the asset itself - ? but even that isn't too clear. Do you mean "restricted to"